pages/kubernetes/how-to/monitor-data-plane-with-cockpit.mdx

```diff
@@ -0,0 +1,169 @@
+
+Because the data plane is entirely under your control, **logs from any components running on these nodes are considered your own data**. Consequently, shipping these logs to Cockpit is billed based on data ingestion.
+
+## How it works
+
+The system leverages **Promtail** (a lightweight log collector) running on your Kapsule or Kosmos cluster. Promtail forwards logs to the Loki endpoint of your Cockpit instance:
+
+1. **Promtail** can collect logs from:
+   - **Container stdout/stderr** (pods)
+   - **systemd journal** (e.g., `kubelet.service`)
+2. **Log data** is transmitted to **Cockpit** (Loki).
+3. **Cockpit** stores and indexes these logs.
+
+## Step-by-step: Enabling container logs in Cockpit
+
+You can use Scaleway’s **[Easy Deploy](/kubernetes/how-to/enable-easy-deploy/)** to add a Promtail deployment to your cluster:
+
+1. **Open the Scaleway Console** and go to your **Kubernetes** cluster.
+2. Navigate to the **Easy Deploy** tab.
+3. Select **Promtail for Cockpit** from the library.
+4. **Deploy** the application. Promtail will install on your cluster with default settings that:
+   - Collect container logs for **all namespaces** (by default).
+   - Collect systemd journal logs (e.g., `kubelet.service`).
+   - Forward logs securely to **Cockpit**.
+    <Message type="note">
+      You can edit the default deployment configuration to filter logs by source (under `config.snippets.scrapeConfigs` in the YAML file). For example:
+      ```yaml
+      cockpit_promtail_scrape_config_pods: "namespace1,namespace2"
+      cockpit_promtail_scrape_config_journal: "kubelet.service,kube-proxy.service"
+      ```
+    </Message>
+### Example Promtail configuration
+Below is a simplified snippet of the configuration that Easy Deploy generates by default:
+```yaml
+config:
+  clients:
+    - bearer_token: "{{{ cockpit_bearer_token }}}" # Automatically set by Easy Deploy
+      url: "{{{ cockpit_loki_push_url }}}" # Automatically set by Easy Deploy
+
+  snippets:
+    scrapeConfigs: |
+      {{{- cockpit_promtail_scrape_config_pods }}} # Default: log all pods
+      {{{- cockpit_promtail_scrape_config_journal }}} # Default: log all system components
+extraVolumeMounts:
+  - mountPath: /var/log/journal
+    name: journal
+    readOnly: true
+extraVolumes:
+  - hostPath:
+      path: /var/log/journal
+    name: journal
+```
+<Message type="note">
+  Template values like `{{{ cockpit_bearer_token }}}` (Bearer Token) and `{{{ cockpit_loki_push_url }}}` (Loki URL) are automatically set. Avoid modifying these values.
+</Message>
+
+## Observing logs in Cockpit
+
+Once Promtail is running:
+
+1. Go to **Cockpit** → **Kubernetes Cluster Pod Logs** (or open your own Grafana connected to Cockpit).
```

Comment:
```suggestion
1. Go to **Cockpit**, then **Kubernetes Cluster Pod Logs** (or open your own Grafana connected to Cockpit).
```

Reply:
"Open your own Grafana connected to Cockpit" => do you mean the Grafana dashboards?

Reply:
yes